tests/examples/DontAwaitInALoopLinter/excluded.hack.expect

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
This is really hard to review... think it's time for a new expect format :) will do that separately

Reply:
In this case you just need to look at the line numbers, e.g. there is an error for line 16 + line 20, but there is no error for line 19 + line 20, as expected because the await on line 20 is inside the loop from line 16, but not inside the loop from line 19.

Reply:
Yeah, it's doable, just not friendly - and not new with this PR, it's just I'm historically the one writing these so not used to being on this end of things :p

https://github.com/hhvm/hhast/issues/263 is one potential approach